### PR TITLE
bug fix+6/1 translation

### PR DESCRIPTION
**Pull request by sealDebolina**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/steps/c100-rebuild/check-your-answers/util/miam.util.ts [hunk 1]
```diff
@@ -90,12 +90,12 @@ export const MiamContentsForDomensticVoilence = UserCase => {
           ...DomesticAbuseCy().letterOfBeingVictim_subFields,
           ...DomesticAbuseCy().letterFromAuthority_subFields,
           ...GeneralContentCy(),
-          generalReasonTitle: GeneralContentEn().title,
```
Comment (vivek-sekhar) [inline src/main/steps/c100-rebuild/check-your-answers/util/miam.util.ts:93]: Removed EN related titles? Where are EN related translations?

File: src/main/steps/c100-rebuild/check-your-answers/mainUtil.ts [hunk 2]
```diff
@@ -293,9 +301,14 @@ export const ChildernDetails = (
...
 export const ChildernDetailsAdditional = (
   { sectionTitles, keys, Yes, No, ...content }: SummaryListContentWithBoolean,
-  userCase: Partial<CaseWithId>
+  userCase: Partial<CaseWithId>,
+  language
 ): SummaryList | undefined => {
-  let htmlForAdditionalText = userCase?.['cd_childrenKnownToSocialServices'];
+  let htmlForAdditionalText='';
+
+//   htmlForAdditionalText =userCase?.['cd_childrenKnownToSocialServices']===YesOrNo.YES?'Ydyn':"Nac Ydyn";
```
Comment (vivek-sekhar) [inline src/main/steps/c100-rebuild/check-your-answers/mainUtil.ts:309]: Remove commented code

File: src/main/steps/c100-rebuild/check-your-answers/mainUtil.ts [hunk 3]
```diff
@@ -1443,4 +1471,92 @@ export const reasonableAdjustment = (
     title: sectionTitles['reasonAbleAdjustment'],
     rows: getSectionSummaryList(SummaryData, content),
   };
-};
\ No newline at end of file
+};
+
+export function genderChose(choice: string, language: string|undefined): string {
```
Comment (vivek-sekhar) [inline src/main/steps/c100-rebuild/check-your-answers/mainUtil.ts:1476]: Can you please remove all below individual functions and put all these translations in content CY translations as separate keys as how I did for recent Yes No translation changes.

**Review by vivek-sekhar — CHANGES_REQUESTED** [review]

File: src/main/steps/c100-rebuild/check-your-answers/util/miam.util.ts (hunk 1 above)
Comment (sealDebolina) [reply src/main/steps/c100-rebuild/check-your-answers/util/miam.util.ts:93]: Enlish content are available at english block. in cy block it was written as english, for that reason some content of check your aswer  screen was not translating

File: src/main/steps/c100-rebuild/check-your-answers/helpers/InternationElementsHelper.ts [hunk 4]
```diff
@@ -10,30 +11,42 @@ const htmlValParser = (selection, subText, keys) => {
...
+        userCase['ie_provideDetailsParents'],
+        keys
+      ),
       changeUrl: Urls['C100_INTERNATIONAL_ELEMENTS_PARENTS'],
     },
     {
       key: keys['anotherPersonSameOrder'],
       valueHtml: htmlValParser(
-        userCase['ie_internationalJurisdiction'],
+        getYesNoTranslation(language, userCase['ie_internationalJurisdiction'], 'gallaiTranslation'),
```
[the remark this answers is not in the document]
Comment (vivek-sekhar) [reply src/main/steps/c100-rebuild/check-your-answers/helpers/InternationElementsHelper.ts:37]: Can you please rename gallaiTranslation to something meaningful?

**Review by vivek-sekhar — APPROVED** [review]

**Pull request merged**